app/grandchallenge/workstation_configs/serializers.py

```diff
@@ -73,6 +73,10 @@ class Meta:
             "default_zoom_scale",
             "show_image_info_plugin",
             "show_display_plugin",
+            "show_invert_tool",
+            "show_flip_tool",
+            "show_window_level_tool",
+            "show_reset_tool",
```

Comment:
Actually, could you add a validator method here as these new things should only be set if `show_display_plugin` is also true.

Reply:
Ok I've had to merge this as is as there is a problem in CIRRUS otherwise.